recipes/r-justifyalpha/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# 'Autobrew' is being used by more and more packages these days
```

Comment:
Please remove all comments in this file (minus the #! line of course).

Reply:
✔️